dali/core/mm/malloc_resource.cc

```diff
@@ -107,6 +108,12 @@ bool cuda_malloc_async_memory_resource::is_supported(int device_id) {
     CUDA_CALL(cudaGetDevice(&device_id));
 
   if (!support[device_id]) {
+    nvml::Init();
+    float driverVersion = nvml::GetDriverVersion();
+    if (driverVersion < 495) {
+      support[device_id] = unsupported;
+      return support[device_id] == supported;
+    }
```

Comment:
This would pessimistically block the use of CUDA malloc async - and Tegra systems don't support NVML, so we would cut them off. I think we should just disable the tests instead.